src/effector/__tests__/event/except.test.ts

```diff
@@ -0,0 +1,26 @@
+import {createEvent} from 'effector'
+import {argumentHistory} from 'effector/fixtures'
+
+describe('event.except', () => {
+  it('filter out when fn returns falsy', () => {
+    const fn = jest.fn()
+    const event = createEvent()
+    const filtered = event.except(x => x > 0)
+    filtered.watch(x => fn(x))
+    event(2)
+    event(-1)
+    expect(argumentHistory(fn)).toEqual([2])
+  })
+  it('works with undefined as well', () => {
+    const fn = jest.fn()
+    const event = createEvent()
+    const filtered = event.filter({
+      fn: x => x === undefined,
+    })
```

Comment:
Probably, it should be except, instead of filter?
```suggestion
    const filtered = event.except(x => x === undefined)
```